frontend/packages/ceph-storage-plugin/src/components/ocs-install/install-page.tsx

```diff
@@ -90,33 +97,39 @@ const InstallCluster: React.FC<InstallClusterProps> = ({ match }) => {
     return sanitizedMode;
   };
 
-  const getStep = (offset: number = 0) => {
-    const searchParams = new URLSearchParams(window.location.search.slice(1));
-    const step = parseInt(searchParams.get('step'), 10) || 1;
-    const sanitizedStep = step && step <= 5 - offset && step >= 1 ? step : 1;
-    return sanitizedStep;
-  };
-
   const getParamString = (step: number, mode: number) => {
     const searchParams = new URLSearchParams(window.location.search.slice(1));
     searchParams.set('step', step.toString());
     searchParams.set('mode', mode.toString());
     return searchParams.toString();
   };
 
+  const getAnchor = (step: number, mode: number) => `~new?${getParamString(step, mode)}`;
+
+  const getStep = (offset: number = 0) => {
+    const searchParams = new URLSearchParams(window.location.search.slice(1));
```

Comment:
```suggestion
    const searchParams = new URLSearchParams(window.location.search);
```
Do we really need to slice it? Are you trying to remove `?` ?

Reply:
I was, but I see it works fine without this as well. Thank you for spotting this!